docs/development/core/development-functional-tests.asciidoc

```diff
@@ -277,47 +314,50 @@ await testSubjects.click(‘containerButton’);
 ** `testSubjects.click(testSubjectSelector)` - Click a test subject in the page; throw if it can't be found after some time
 
 **find:**:::
-* Source: {blob}test/functional/services/find.js[test/functional/services/find.js]
+* Source: {blob}test/functional/services/find.ts[test/functional/services/find.ts]
 * Helpers for `remote.findBy*` methods that log and manage timeouts
 * Popular methods:
 ** `find.byCssSelector()`
 ** `find.allByCssSelector()`
 
+**retry:**:::
+* Source: {blob}test/common/services/retry/retry.ts[test/common/services/retry/retry.ts]
+* Helpers for retrying operations
+* Popular methods:
+** `retry.try(fn, onFailureBlock)` - execute `fn` in a loop until it succeeds or the default try timeout elapses, optional `onFailureBlock` is executed before each retry attempt 
+** `retry.tryForTime(ms, fn, onFailureBlock)` execute `fn` in a loop until it succeeds or `ms` milliseconds elapses, , optional `onFailureBlock` is executed before each retry attempt
```

Comment:
For consistency, I would change this to:

- Execute `fn` in a loop until it succeeds or `ms` milliseconds elapses. The optional `onFailureBlock` is executed before each retry attempt.